helios-services/src/main/java/com/spotify/helios/agent/AddExtraHostContainerDecorator.java

```diff
@@ -17,6 +17,7 @@
 
 package com.spotify.helios.agent;
 
+import com.google.common.base.Optional;
```

Comment:
So then we have a mix of both optional types throughout the code? Don't think that's a good idea. If we want to switch to `java.util.Optional` everywhere, that should be separate.